Project-1/Praoject_1_final_version

```diff
@@ -0,0 +1,384 @@
+    John : B
+    Chris : M
+    Mary : ¬M
+    Mary: A
+    Alex : parent_of.Mary
+    Alex : parent_of.John
+    }
+
+  ΔI = {John, Mary, Alex, Chris}
+
+  Named Individuals:
+  JohnI = J,
+  MaryI = M,
+  AlexI = A,
+  Chris = C
+
+  Concept Assignments:
+  PersonI = {J, M, A, C},
+  MI = {J, A, C},
+  FI = {M},
+  BI = {J},
+  AI = {M}
+
+Role Assignments:
+parentofI = {(J, C), (A, J), (A, M)}
+
+  ```
+6. Explain the difference - using natural language - between the first-order prefixes:
+  ```
+  (a) ∃x∀y and ∀x∃y
+	For statements coming after ∃x∀y, there must be an x included in them while there is no necessity of them having a y. Also, for such statements to be true, they must be true in all conditions of y (T or F), but as long as they are true for at least one case of x they are fine.
+	For statements coming after ∀x∃y, there must be a y included in them while there is no necessity of them having an x. Also, for such statements to be true, they must be true in all conditions of x (T or F), but as long as they are true for at least one case of y they are fine.
+	
+  (b) ∃x∀y∃z and ∀x∃y∀z 
+  	For statements coming after ∃x∀y∃z, there must be an x and a z included in them while there is no necessity of them having a y. Also, for such statements to be true, they must be true in all conditions of y (T or F), but as long as they are true for at least one case of x and one case of z they are fine.
+	For statements coming after ∀x∃y∀z, there must be a y included in them while there is no necessity of them having an x or a y. Also, for such statements to  be true, they must be true in all conditions of x and all conditions of z (T or F), but as long as they are true for at least one case of y they are fine.
+	
+  (c) ∀x∃y∀z∃w and ∃x∀y∃z∀w
+  	For statements coming after ∀x∃y∀z∃w, there must be a y and a w included in them while there is no necessity of them having an x or z. Also, for such statement to be true, they must be true in all conditions of x and all conditions of z (T or F), but as long as theyr are true for at least one case of y and one case of w they are fine.
+	For statements coming after ∃x∀y∃z∀w, there must be an x and a w included in them while there is no necessity of them having a y or w. Also, for such statement to be true, they must be true in all conditions of y and all conditions of w (T or F), but as long as theyr are true for at least one case of x and one case of z they are fine.
+```
+	
+7. Show that the following sentences are not equivalent by exhibiting a graph that models one but not both of these sentences:
+```
+∀x∃y∀z(R(x,y) ∧ R(x,z) ∧ R(y,z))
+∃x∀y∃z(R(x,y) ∧ R(x,z) ∧ R(y,z))
+
+{(a,b),(b,a),(a,a)}
+picture was not uploaded
+
+```
+	
+8. Using an online tableau proof generator - such as the one found here `https://www.umsu.de/trees/` - provide tree proofs of the following entailments, which are known as the De Morgan's laws:
+  ```
+  (a) ∀x∀y(¬(Px ∧ Qx) → (¬Px ∨ ¬Qx))
+  
+	1.	¬∀x∀y(¬(Px ∧ Qx) → (¬Px ∨ ¬Qx))
```

Comment:
What rule justifies going from ¬∀x∀y to ¬∀y i.e. instantiating the first universal while it is within the scope of the first negation?